portfolio/src/main/webapp/index.html

```diff
@@ -14,7 +14,12 @@
 
       <header></header>
 
-      <body>
+      <body onload="getData()">
+
+        <!-- getData() will fill out this box when present. Currently, we don't need this for a few more steps
```

Comment:
It is better to avoid commented out code, add a TODO instead.

Reply:
I didn't know that, thanks!